### DISCUSSION TITLE
Failed Installation

### DISCUSSION THREAD

**Discussion by M4jx** [Security-Onion-Solutions/securityonion#12806] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

32

### Storage for /

69.2

### Storage for /nsm

133.7

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Trying to perform a fresh install fails at the end of the setup. I have tried to perform the installation multiple times, however, it is always failing at the end of the installation and returning the same failure message in errors.log.

**errors.log**
```
[ERROR   ] {'container': {'Networks': {'bridge': {'IPConfiguration': {'old': 'automatic', 'new': 'not connected'}}, 'sobridge': {'Aliases': {'old': None, 'new': ['c1d9d0184051']}, 'IPAMConfig': {'old': None, 'new': {'IPv4Address': '172.17.1.25'}}}}}, 'container_id': {'added': 'c1d9d0184051d7cc1890fe1a44e12be84703e46a1612dac88b1b79f3c1ebc144'}}
[ERROR   ] {'container': {'Networks': {'bridge': {'IPConfiguration': {'old': 'automatic', 'new': 'not connected'}}, 'sobridge': {'Aliases': {'old': None, 'new': ['06bcdbcf3984']}, 'IPAMConfig': {'old': None, 'new': {'IPv4Address': '172.17.1.30'}}}}}, 'container_id': {'added': '06bcdbcf3984bf452cbd81a11d82db43c38affc8eeb7e2d12b7d30f288460935'}}
      Result: False
     Comment: Created container 'so-idstools'. Failed to start container 'so-idstools': 'Unable to perform start: UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)'.
      Result: False
     Comment: Created container 'so-mysql'. Failed to start container 'so-mysql': 'Unable to perform start: UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)'.
      Result: False
     Comment: One or more requisite failed: idstools.enabled.so-idstools
[ERROR   ] Command '/opt/so/__salt.tmp.o1uiy0q5' failed with return code: 1
[ERROR   ] stdout: 
[ERROR   ] stderr: 
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 267235, 'retcode': 1, 'stdout': '\nInstallation initiated, view install log for further details.', 'stderr': '\n\nInstalling in non-interactive mode.: signal: killed'}
      Result: False
     Comment: One or more requisite failed: ssl.influxdb_crt
      Result: False
```

After the failure message is displayed, the web interface, dashboard, and alerts are working just fine. However, not sure what causes this error, or if it affects a feature that I'm not currently using.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
The `/root/sosetup.log `would be better if you can share, but please run `sudo so-checkin` and let me know if you see any errors.

**Reply by M4jx** [reply]
**so-checkin**
```
[root@so-standalone admin]# so-checkin
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/grains'
[INFO    ] Syncing grains for environment 'base'
[INFO    ] Loading cache from salt://_grains, for base
[INFO    ] Caching directory '_grains/' for environment 'base'
local:
    Data failed to compile:
----------
    The function "state.highstate" is running as PID 3629457 and was started at 2024, Apr 17 19:57:09.439411 with jid 20240417195709439411
```

**/root/sosetup.log** (couldn't paste it here due to it being larger than what Github allows)
https://file.io/8cZWOG2KLbVD

**Reply by cm-ops** [reply]
That command just means a highstate was running when you issue the `so-checkin` command. You could run `sudo salt-call state.highstate -l info queue=True` then the command will wait if another highstate is running before executing.
